### PR TITLE
remove network dependency for unit tests

### LINKED ISSUE #47: Remove internet connection requirement for running tests
Tests make use of CDN hosted versions of jQuery and Keen.js. Ideally, you shouldn't need an internet connection to run tests.

### PR DESCRIPTION
**Pull request by josephwegner**
closes #47 

This moves keen-js and jquery dependencies into NPM, so that tests can load them without a network connection.

### CODE REVIEW

File: test/unit/index.js [hunk 1]
```diff
@@ -1,5 +1,11 @@
 /** @jsx React.DOM */
 
+// **********************
+// Dependencies
+// **********************
+require('keen-js');
+window.$ = window.jQuery = require('jquery-browserify');
```
Comment (joannecheng) [inline test/unit/index.js]: is this line needed? The tests seem to work with just `require('jquery-browserify')`.

Comment (josephwegner) [inline test/unit/index.js]: You're probably right. I was just following their README :)

**Comment by josephwegner** [comment]
cleaned up

**Comment by joannecheng** [comment]
:+1:

**Pull request merged**